jupyter/datascience/anaconda-python-3.8/Dockerfile

```diff
@@ -0,0 +1,57 @@
+ARG BASE_IMAGE
+FROM ${BASE_IMAGE}
+
+LABEL name="s2i-minimal-notebook-anaconda:latest" \
+    summary="Minimal Jupyter Notebook Source-to-Image for Python 3.8 applications. Uses Anaconda CE instead of pip." \
+    description="Notebook image based on Anaconda CE Python.These images can be used in OpenDatahub JupterHub." \
+    io.k8s.description="Notebook image based on Anaconda CE Python.These images can be used in OpenDatahub JupterHub." \
+    io.k8s.display-name="Anaconda s2i-minimal-notebook, python38" \
+    io.openshift.expose-services="8888:http" \
+    io.openshift.tags="python,python38,python-38,anaconda-python38" \
+    authoritative-source-url="https://quay.io/modh/s2i-minimal-notebook-anaconda" \
+    io.openshift.s2i.build.commit.ref="master" \
+    io.openshift.s2i.build.source-location="https://github.com/red-hat-data-services/s2i-minimal-notebook-anaconda" \
+    io.openshift.s2i.build.image="quay.io/modh/s2i-minimal-notebook-anaconda:latest"
```

Comment:
Configure this registry for the downstream. 
For the upstream mirror the image under the `quay.io/repository/opendatahub/workbench-images`